src/goal.rs

```diff
@@ -17,16 +19,12 @@ pub struct Goal {
 	/// A goal's description
 	pub description: String,
 	/// How much total time should a user put into their goal, eg "I want to learn how to code, and I want to code 6 hours per day"
-	pub task_duration: time::Duration,
+	pub task_duration: Duration,
```

Comment:
```suggestion
	pub duration: Duration,
```
Goal has a duration, not a task_duration